src/server/service/match.ts

```diff
@@ -305,8 +321,7 @@ export class Match {
 
     this.arePointsCalculated = true;
   }
-
-  async storeScore(tx?: BBPgTransaction) {
+  async storeMatchStats(tx?: BBPgTransaction) {
     const dbTx = tx ?? db;
```

Comment:
At this point I'd just make the tx to be a mandatory param since you are doing more calls in this function and you want them to be atomic